front/deploypackage.public.php

```diff
@@ -63,10 +63,12 @@
    //If it's a local wakeup, local call to the agent RPC service
     switch ($_POST['wakeup_type']) {
         case 'local':
-            echo '<link rel="import" href="http://127.0.0.1:62354/now">';
-            echo Html::scriptBlock("setTimeout(function(){
-            window.location='{$_SERVER['HTTP_REFERER']}';
-         }, 500);");
+            echo Html::scriptBlock("
+                $.get('http://127.0.0.1:62354/now');
```

Comment:
Agent port should if possible retrieved from the agent object.

Reply:
I see no way to find the agent

Reply:
Will be hard to achieve in this 'local' case as it doesn't target a specific computer, but the localhost only.
I need to find the good computer for the current user, and if he has more than one, I will not be able to find the good one.

The better I can do is retrieving the default agent port.

Reply:
Lines 53 to 60, I see a loop which purpose seems to retrieve the `$computers_id`. Is really there no way to retrieve the related agent from this id ? And if unknown, yes, using the default configured agent port would be nice.

Reply:
An even, there's an example in the "remote" case. For me something like this works:
```php
            if ($computers_id) {
                $agent = new Agent();
                $agent->getFromDBByCrit(['itemtype' => 'Computer', 'items_id' => $computers_id]);
                $port = (int)$agent->fields['port'];
            }
            echo Html::scriptBlock("
                $.get('http://127.0.0.1:".(isset($port) && $port > 0 ? $port : 62354)."/now');
                setTimeout(function(){
                    window.location='{$_SERVER['HTTP_REFERER']}';
                }, 500);
            ");
```